SB2018/2018-05-22.md

```diff
@@ -0,0 +1,26 @@
+---
+to: All Teams
+subject: A massive thank you! And a request for feedback
+---
+
+![SB2018 team photo](https://scontent.flhr3-2.fna.fbcdn.net/v/t1.0-9/32407350_209412883122668_9114781927738441728_n.jpg?_nc_cat=0&oh=847f2cf2b70746f963e88de8e77cf4dc&oe=5B84A368)
+
+Hello!
+
+First of all, a massive thank you for making SourceBots 2018, our first large-scale public event, such a success!
+
+An article about the competition, results, and game has been [posted on our website](https://sourcebots.co.uk/blog/2018-04-23-noo-noo-from-collyers-college-wins-sourcebots-2018/), along with a [breakdown of the scores](https://sourcebots.co.uk/events/2018/competition#league-scores). Our photos of the event have also been uploaded to our [Facebook page](https://www.facebook.com/pg/SourceBots/photos/?tab=album&album_id=207232630007360).
+
+# Feedback
+
+We'd love to get some feedback on what went well, and what could be improved next time. It would be greatly appreciated if you, your team, and anyone else who came with you could fill out [our form](https://goo.gl/forms/AcLCRfI5iF0GnLM02). All feedback is given anonymously.
+
+# Kit Return
+A reminder to teams who still have their kits. We need the kits back in Southampton by the 1st June at the latest. If you haven't already, please arrange shipment to:
+
+> Student Robotics Society\
```

Comment:
This is previously Student Robotics (Southampton) and will be changing soon to Southampton Robotics Outreach

Reply:
As this is the current name of the society, it's correct